indico_toolkit/association/association.py

```diff
@@ -0,0 +1,59 @@
+from typing import Union, List, Dict
+from collections import defaultdict
+from indico_toolkit.types import Predictions
+
+
+class Association:
```

Comment:
should we think about using abc to make abstract base classes more explicit?

Reply:
added this